radp/digital_twin/mobility/ue_tracks_params.py

```diff
@@ -0,0 +1,174 @@
+from typing import Dict
+
+from radp.common import constants
+from radp.digital_twin.mobility.ue_tracks import MobilityClass
+from services.ue_tracks_generation.ue_tracks_generation_helper import (
+    UETracksGenerationHelper,
+)
+
+
+class UETracksGenerationParams:
```

Comment:
add a unit test

Reply:
Added the Unit Test Cases for the ue_tracks_params.
All of the changes are made.

Reply:
You check the second last commit to see the changes made in mobility/test module. The last commit was done to format all of the codes using Black.